net-misc/bird/files/bird-2.0.9-musl-tests.patch

```diff
@@ -0,0 +1,40 @@
+From 0f679438f36d8c2a31dfe490007e983b085caef6 Mon Sep 17 00:00:00 2001
```

Comment:
Link to the commit / ML post if you can too

Reply:
I’ve added the two links